src/sensor/run_sensor.sh

```diff
@@ -0,0 +1 @@
+python3 run_sensor.py >> logs/sensor.log &
```

Comment:
Przerzuć te plik do /scripts/. Wtedy będizemy mieli wszystkie skrypty potrzebne do uruchomienia w jednym katalogu i logi przez to też wyladują w jednej lokalizacji.